.github/workflows/canary-integration-test.yml

```diff
@@ -241,6 +265,17 @@ jobs:
       run: |
         tests/scripts/validate_cluster.sh osd
         kubectl -n rook-ceph get pods
+        kubectl -n rook-ceph get pods --field-selector=status.phase==Running  | sed '/^csi/d' | sed -n '1!p' | awk '{print $1}' > pods.txt
+        kubectl -n rook-ceph get pods | grep -i "rook-ceph-osd-prepare" | awk '{print $1}' >> pods.txt
+        mkdir test
+        cat pods.txt | while read line; do kubectl -n rook-ceph logs $line > test/$line ; done
+
+    - name: Upload pvc-db test result
+      uses: actions/upload-artifact@v2
+      if: always()
```

Comment:
If it's always executing, can we remove this line?

Reply:
In case of success it doesn't upload artifact that's why I added `if: always()`